hogan-form.php

```diff
@@ -22,14 +22,10 @@
 
 require_once 'class-form.php';
 
-// Only load module if dependant plugins are active.
-if ( function_exists( 'gravity_form' ) && class_exists( 'ACFGravityformsField\Field' ) ) {
+add_action( 'plugins_loaded', function () {
```

Comment:
Skriv gjerne om de to closure-funksjonene i `hogan-form.php` til funksjoner i global namespace slik at det er mulig å avregistrere modulen. Se https://github.com/DekodeInteraktiv/hogan-text/blob/master/hogan-text.php#L31